node/views/bmi-calculator/bmi-calculator.ejs

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+    <title>BMI Calculator</title>
+    <link rel="stylesheet" href="./stylesheets/bmi-calc.css">
+</head>
+proj
+
+<body>
+    <div>
+       <h2>BMI Calculator</h2>
+        <p class="text">Height</p>
+        <input type="text" id="h">
+        <p class="text">Weight</p>
+        <input type="text" id="w">
+        <p id="result"></p>
+        <p id="status"></p>
+        <p id="suggestion"></p>
+        <button id="btn">Calculate</button>
```

Comment:
btn or button is fine for class but it is too generic for an id. Their can be multiple button in a page